test/Base.t.sol

```diff
@@ -34,6 +35,7 @@ abstract contract Base_Lending_Test is Base_Test, Events, Errors {
     TrancheExtension internal jrTranche;
     TrancheExtension internal srTranche;
     TrancheExtension internal tranche;
+    TrancheWrapper internal trancheWrapper;
```

Comment:
No need to add it to base.sol (-> will be deployed for every test in this repo).
Only declare it in _TrancheWrapper.sol